site/src/SUMMARY.md

```diff
@@ -17,6 +17,7 @@
   - [Other options](book/other-options.md)
   - [Environment variables](book/env-vars.md)
   - [Stability policy](book/stability.md)
+  - [Reusing build](book/reusing-build.md)
```

Comment:
Could you remove this line for now? We'll add it after the new version of nextest is released.